### PR TITLE
Edited several tags

### PR DESCRIPTION
**Pull request by Keyacom**
- `environments.md`:
  - Added `envs` alias
  - Moved `Python Environments` to the embed's title field
  - Unwrapped `sys.executable` since it's a REPL example
- `identity.md`: Converted the example with lists to a snippet from a REPL session.
- `inline.md`: Added info about including a backtick in an inline code snippet.
- `listcomps.md`: `if` in list comprehensions is an expression, not a statement, so I changed that.
- `local-file.md`: Short explanation of `rb`, changed modal verb in a sentence
- `slicing.md`: The word "slicing" at the start is now in bold, not in italics.
- `traceback.md`: Added a line that appears since newer Python versions in tracebacks.
- `venv.md`:
  - Added `virtualenv` alias
  - Moved `Virtual Environments` to the embed's title
  - PowerShell is no longer called Windows PowerShell, since its core is cross-platform now. Instead, mentioned that it's about PowerShell in Windows.
  - `-Scope CurrentUser` will make the change apply just for the user, so mentioned that it's required *once per user*.
  - Changed the PowerShell snippet to make it have syntax highlighting

I felt like changing it since I saw what I could improve in `venv.md`, but decided to check more tags as well.

### CODE REVIEW

**Review by shtlrs — APPROVED** [review]
Neat!
Thanks :D

File: bot/resources/tags/listcomps.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ Using list comprehensions can make this both shorter and more readable. As a lis
 >>> [n ** 2 for n in range(5)]
 [0, 1, 4, 9, 16]
 ```
-List comprehensions also get an `if` statement:
+List comprehensions also get an `if` expression:
```
Comment (CaedenPH) [inline bot/resources/tags/listcomps.md:13]: Why this change?

Comment (Keyacom) [reply bot/resources/tags/listcomps.md:13]: Statements must be separated by newlines or semicolons, that's why.

Comment (MarkKoz) [reply bot/resources/tags/listcomps.md:13]: Neither statement nor expression seem like appropriate descriptions since the `if` is not a standalone thing in the context of a comprehension. The [documentation](https://docs.python.org/3/reference/expressions.html#displays-for-lists-sets-and-dictionaries) calls them "`if` clauses".

Comment (Keyacom) [reply bot/resources/tags/listcomps.md:13]: I could modify so it's "clause", not "expression" nor "statement".

File: bot/resources/tags/slicing.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ aliases: ["slice", "seqslice", "seqslicing", "sequence-slice", "sequence-slicing
 embed:
     title: "Sequence slicing"
 ---
-*Slicing* is a way of accessing a part of a sequence by specifying a start, stop, and step. As with normal indexing, negative numbers can be used to count backwards.
+**Slicing** is a way of accessing a part of a sequence by specifying a start, stop, and step. As with normal indexing, negative numbers can be used to count backwards.
```
Comment (wookie184) [inline bot/resources/tags/slicing.md:6]: I think italics was fine here personally, is there a reason for this change?

**Review by wookie184 — COMMENTED** [review]
Thanks for this, the changes look good.

Side note: I didn't realise we have two tags for virtual environments, maybe we should combine them into one (not in this PR)

File: bot/resources/tags/slicing.md (hunk 2 above)
Comment (Keyacom) [reply bot/resources/tags/slicing.md:6]: Consistency with other tags.

**Review by CaedenPH — APPROVED** [review]

**Comment by shtlrs** [comment]
> Side note: I didn't realise we have two tags for virtual environments, maybe we should combine them into one (not in this PR)

I think that's what Tizzy meant by opening #1835

**Review by wookie184 — APPROVED** [review]
